idea/src/org/jetbrains/kotlin/idea/run/JetRunConfigurationEditor.java

```diff
@@ -61,6 +64,8 @@ protected void applyEditorTo(JetRunConfiguration configuration) throws Configura
         myModuleSelector.applyTo(configuration);
         myCommonProgramParameters.applyTo(configuration);
         configuration.setRunClass(myMainClassField.getText());
+        configuration.setAlternativeJrePath(alternativeJREPanel.getPath());
+        configuration.setAlternativeJrePathEnabled(alternativeJREPanel.isPathEnabled());
```

Comment:
Yes. JetRunConfiguration already implements CommonJavaRunConfigurationParameters and exposes those values through that API to the shared code in IDEA.